adapters/Line/LineAdapter/GADMediationAdapterLineUtils.m

```diff
@@ -82,9 +85,33 @@
   }
   [config setNeedChildDirectedTreatment:needChildDirectedTreatment];
   [FADSettings registerConfig:config];
+  gRegisteredFADConfig = config;
```

Comment:
`[FADAdLoader adLoaderForConfig:error:]` is compatible with `[FADSettings registerConfig:]`.

We recommend you to use `[FADAdLoader adLoaderForConfig:error:]` and save the return value of the `[FADAdLoader adLoaderForConfig:error:]` instead of `FADConfig`.

---

`[FADAdLoader adLoaderForConfig:error:]` は `[FADSettings registerConfig:]` の上位互換になっています。

`[FADSettings registerConfig:]` の代わりに `[FADAdLoader adLoaderForConfig:error:]` を利用し、`FADConfig` ではなく `[FADAdLoader adLoaderForConfig:error:]` の返り値を保存して利用することをお勧めします。

Reply:
Done!